### ISSUE TITLE
Run script after /opt/run

### ISSUE THREAD

**Issue by jtonello** [jjethwa/icinga2#211]
I'd like to run a simple bash configuration script after the icinga2 service is running in the primary container (post /opt/run), but I've not been successful adding a CMD or ENTRYPOINT in the Dockerfile. Any suggestions?

**Comment by jjethwa** [comment]
Hi @jtonello 

There is the custom_run script: https://github.com/jjethwa/icinga2/blob/master/content/opt/setup/10-customrun 

You mentioned you need the icinga2 service running though?  Can you give me an idea what you are trying to do?

**Comment by jtonello** [comment]
Hi @jjethwa,

I have a simple script that scans the network and auto-creates the
hosts.conf objects. The last steps are to move the file to
/etc/icinga2/conf.d/ and reload icinga2. I couldn't figure out how to
trigger it after /opt/run.

**Comment by jjethwa** [comment]
Hi @jtonello 

That definitely sounds like something you can use the custom run script to do.  Try adding your script using a volume as /opt/custom_run

**Comment by jtonello** [comment]
I placed my script as /opt/custom_run, but I'm thinking that's not quite
right?

**Comment by jjethwa** [comment]
Hi @jtonello 

So the script is named custom_run and under /opt, right?

**Comment by jtonello** [comment]
Yes. Do I need to make another entrypoint or does the /opt/run entrypoint
take care of that?

**Comment by jjethwa** [comment]
Hi @jtonello 

No, you don't need to:

https://github.com/jjethwa/icinga2/blob/master/content/opt/run#L45

https://github.com/jjethwa/icinga2/blob/master/content/opt/setup/10-customrun

If your script has output, you should see it during the container run.

**Comment by jtonello** [comment]
The script is running, but the issue appears to be that my script wants to
write a couple temp files and it errors out, saying it can't find them.
I've tried placing them in / /tmp and even the /opt folders.

**Comment by jjethwa** [comment]
Hi @jtonello 

The script says it's unable to write to /tmp?  Is it running as a different user?
